arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -951,6 +951,13 @@ def test_plot_posterior(models, kwargs):
         assert axes.shape
 
 
+def test_plot_posterior_boolean():
+    data = np.random.choice(a=[False, True], size=(4, 100))
+    axes = plot_posterior(data)
+    assert axes
+    assert axes.get_xticklabels()[0].get_text() != 0
```

Comment:
I know this is not the best way to see if xticklabels are correct. It should be like `assert axes.get_xticklabels()[0].get_text() == 'False'. 
When I tried it locally, tests work well locally only if I provide `show=True` to the plot function call and shows an assertion error when show=False (default).
`AssertionError: assert ' ' == 'False'`
Azure jobs fails with same error even if `show=True` is pushed.

Reply:
maybe it could be solved using some non-interactive/directly writing to file backend? I don't reallly know what is happening, but it would not be surprising that Azure had some extra checks preventing guis, pop ups or other interactive objects

Reply:
Already tried using `plt.draw()` in the matplotlib backend file, didn't work! I guess we need to call `plt.draw()` in the test function before assert or maybe leave it like that if its an "okayish" way to test if not good.